File: docs/datatools/codesnippet/VisualBasic/how-to--execute-a-stored-procedure-that-returns-no-value_2.vb
        Dim sqlConnection1 As New SqlConnection("Your Connection String")
        Dim cmd As New SqlCommand
        Dim rowsAffected As Integer

        cmd.CommandText = "StoredProcedureName"
        cmd.CommandType = CommandType.StoredProcedure
        cmd.Connection = sqlConnection1

        sqlConnection1.Open()

        rowsAffected = cmd.ExecuteNonQuery()

        sqlConnection1.Close()

File: docs-old/datatools/codesnippet/CSharp/how-to--execute-a-stored-procedure-that-returns-no-value_2.cs
            SqlConnection sqlConnection1 = new SqlConnection("Your Connection String");
            SqlCommand cmd = new SqlCommand();
            Int32 rowsAffected;
            
            cmd.CommandText = "StoredProcedureName";
            cmd.CommandType = CommandType.StoredProcedure;
            cmd.Connection = sqlConnection1;

            sqlConnection1.Open();

            rowsAffected = cmd.ExecuteNonQuery();

            sqlConnection1.Close();